lib/charms/layer/jenkins/configuration.py

```diff
@@ -17,12 +17,16 @@ def bootstrap(self):
         hookenv.log("Bootstrapping initial Jenkins configuration")
 
         config = hookenv.config()
-        context = {"master_executors": config["master-executors"]}
+        context = {
+            "master_executors": config["master-executors"],
+            "jnlp_port": config["jnlp-port"],
+        }
         templating.render(
             "jenkins-config.xml", paths.CONFIG_FILE, context,
             owner="jenkins", group="nogroup")
 
         hookenv.open_port(PORT)
```

Comment:
This line should be removed. Also the PORT global variable can be dropped as well, since it shouldn't be used anymore.